models/game-summary/buildEnhancedGameSummary.js

```diff
@@ -135,13 +136,15 @@ function buildEnhancedGameSummary(_summary) {
 	// Option[List[Option[{ ja: Boolean, presidentId: Int, chancellorId: Int }]]]
 	const votesOf = username => {
 		return indexOf(username).map(i =>
-			turns.map(t =>
-				t.votes.get(i).map(v => ({
-					ja: v,
-					presidentId: t.presidentId,
-					chancellorId: t.chancellorId
-				}))
-			)
+			turns
+				.filter(t => t.votes.get(i))

```

Comment:
I'm not sure why, but I had to add this line or none of the games could create replays, even from the base v1.7.2 code. 🤔